### ISSUE TITLE
tests with fixtures

### ISSUE THREAD

**Issue by benmccann** [svelte-add/svelte-add#461]
### Topic

all

### Description

The tailwind adder is broken right now (https://github.com/svelte-add/svelte-add/issues/459). It'd be nice if we had tests that could catch that before a PR gets merged

### Describe the solution you'd like

Use vitest fixtures. Check that the generated project matches what is expected. That will also help us in code review because we can look at the generated project and see what we think about the diff. We can also setup a script in the package.json to automatically update the fixtures so that they're not hard to maintain when we make changes to the code

**Comment by benmccann** [comment]
oh, I just noticed that this is a bit of a duplicate of https://github.com/svelte-add/svelte-add/issues/356

**Comment by manuel3108** [comment]
I don't think that it really is. The other issue would just make sure that `Welcome to Sveltkit` (or whatever) is rendered. But this would be way better for code review and so one. 

The only thing that makes me re-consider this is making `vitest` work with the adders. I tried this in the past, and failed. But we might have a better chance now that #446 is implemented.

**Comment by manuel3108** [comment]
Is it possible that you mean [`snapshots`](https://vitest.dev/guide/snapshot.html)? I could not find much about [`fixtures`](https://vitest.dev/guide/test-context.html#fixture-initialization) in their docs.
